src/MOIWrapper.jl

```diff
@@ -0,0 +1,561 @@
+        append_row(row, lower, upper, instance, rows, cols, coefs)                
+    end
+end
+
+
+"""
+    get_rhs(instance, row::Int)::Float64
+
+Get the right-hand side of the linear constraint in the 1-indexed row `row` in
+the model `m`.
+"""
+function LQOI.get_rhs(instance::ClpOptimizer, row::Int)::Float64
+    lower = replace_inf(get_row_lower(instance.inner))
+    upper = replace_inf(get_row_upper(instance.inner))
+    lb = lower[row]
+    ub = upper[row]
+    
+    if lb > -Inf
+        return lb
+    elseif ub < Inf
+        return ub
+    else
+        error("Either row_lower or row_upper must be of abs less than 1e20")
+    end
+end
+
+"""
+    get_linear_constraint(instance, row::Int)::Tuple{Vector{Int}, Vector{Float64}}
+
+Get the linear component of the constraint in the 1-indexed row `row` in
+the model `m`. Returns a tuple of `(cols, vals)`.
+"""
+function LQOI.get_linear_constraint(instance::ClpOptimizer, row::Int)::Tuple{Vector{Int}, Vector{Float64}}
+    A = get_constraint_matrix(instance.inner)
+    A_row = A[row,:]
+    return (Array{Int}(A_row.nzind), A_row.nzval)
+end
+
+"""
+    change_objective_coefficient!(instance, col, coef)
+
+Set the linear coefficient of the variable in column `col` to `coef` in the objective function.
+"""
+function LQOI.change_objective_coefficient!(instance::ClpOptimizer, col, coef)
+    objcoefs = get_obj_coefficients(instance.inner)
+    objcoefs[col] = coef
+    chg_obj_coefficients(instance.inner, objcoefs)
+end
+
+"""
+    change_rhs_coefficient!(instance, row, coef)
+
+Set the rhs of the constraint in row `row` to `coef`.
+"""
+function LQOI.change_rhs_coefficient!(instance::ClpOptimizer, row, coef)
+    lower = replace_inf(get_row_lower(instance.inner))
+    upper = replace_inf(get_row_upper(instance.inner))
+    lb = lower[row]
+    ub = upper[row]
+    
+    if lb > -Inf
+        lower[row] = coef
+        chg_row_lower(instance.inner, lower)
+    elseif ub < Inf
+        upper[row] = coef
+        chg_row_upper(instance.inner, upper)
+    else
+        error("Either row_lower or row_upper must be of abs less than 1e20")
+    end
+end
+
+ 
+"""
+    delete_linear_constraints!(instance, start_row::Int, end_row::Int)::Void
+
+Delete the linear constraints `start_row`, `start_row+1`, ..., `end_row` from
+the model `m`.
+"""
+function LQOI.delete_linear_constraints!(instance::ClpOptimizer, start_row::Int, end_row::Int)::Void    
+    which = [Int32(i-1) for i in start_row:end_row]
+    delete_rows(instance.inner, which)
+end
+
+
+"""
+    change_linear_constraint_sense!(instance, rows::Vector{Int}, sense::Vector{Cchar})::Void
+
+Change the sense of the linear constraints in `rows` to `sense`.
+"""
+function LQOI.change_linear_constraint_sense!(instance::ClpOptimizer, rows::Vector{Int}, sense::Vector{Cchar} )::Void
+    lower = replace_inf(get_row_lower(instance.inner))
+    upper = replace_inf(get_row_upper(instance.inner))
+        
+    for (i,row) in enumerate(rows)                
```

Comment:
If you use a zip you don't have to access the `i`'th element. 

`for (sense, row) in zip(senses, rows)`